server/endpoint_urls/events_api_url/events_api_url.go

```diff
@@ -15,3 +15,12 @@ func WithPath(path string) *url.URL {
 func String() string {
 	return eventsAPIURL.String()
 }
+
+func URL() *url.URL {
+	if eventsAPIURL == nil {
+		return nil
+	}
+	// Make a copy so the caller can't accidentally modify the original.
+	u := *eventsAPIURL
```

Comment:
IIUC `eventsAPIURL` is a native gRPC endpoint (e.g. remote.buildbuddy.io) and the app doesn't speak gRPC, so I think the app shouldn't need to communicate with this URL?

Reply:
When I visit https://app.buildbuddy.dev and go to `Executions`, I get CSP violations for `https://events.buildbuddy.dev/rpc/...`. Is that unexpected?

Reply:
(I think you meant `Executors`? I was able to repro it there but not the Executions tab)

I think it's because of this "Test region" stuff - @siggisim can we remove this?

https://github.com/buildbuddy-io/buildbuddy-internal/blob/446e52484f833a13c42ea063e1e2c8b13be61015/enterprise/config/buildbuddy.dev.yaml#L284-L286

Reply:
Yeah we can remove that

Reply:
Alternatively, I think the correct thing to do is to look at that "regions" config and allowlist each of the configured subdomains.